tests/API/APIRestTest.php

```diff
@@ -423,18 +441,20 @@ public function testBadEndpoint($session_token) {
          $res = $this->doHttpRequest('GET', 'badEndpoint/',
                                             ['headers' => [
                                              'Session-Token' => $session_token]]);
+         $this->assertFalse(true); // if HTTP 2xx then the test must fail
```

Comment:
i think i would prefer 

$this->assertLessThan(300, $response->getStatusCode());